perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -226,6 +250,28 @@ def ApiBasedBenchmarks(results, metadata, vm, storage, test_script_path,
       # User only wants to run the CLI based tests, do nothing here:
       return
 
+    def BuildCommandString(args):
```

Comment:
If we don't foresee a reason for other code to call BuildCommandString() outside ApiBasedBenchmarks() then I think it's OK to keep it in here.  

As to naming, since it's building a command string specifically for calling our object_storage_test_script, so let's make sure the name reflects that. ;)